File: executable/julia/2d-histogram.jl
using Plotly
Plotly.signin("test-runner", "9h29fe3l0x")

x = randn(500)
y = randn(500)+1

data = [
  [
    "x" => x, 
    "y" => y, 
    "type" => "histogram2d"
  ]
]

response = Plotly.plot([data], ["filename" => "2d-histogram", "fileopt" => "overwrite"])
plot_url = response["url"]

File: executable/python/2d-histogram.py
import plotly.plotly as py
from plotly.graph_objs import *

py.sign_in('test-runner', '9h29fe3l0x')

import numpy as np

x = np.random.randn(500)
y = np.random.randn(500)+1

data = Data([
    Histogram2d(
        x=x,
        y=y
    )
])

plot_url = py.plot(data, filename='2d-histogram')